test/dlc_tests/eks/eks_manifest_templates/pytorch/training/multi_node_gpu_training.yaml

```diff
@@ -33,4 +35,6 @@ spec:
               - <EPOCHS>
             resources:
               limits:
-                nvidia.com/gpu: <GPU_LIMIT>
\ No newline at end of file
+                nvidia.com/gpu: <GPU_LIMIT>
+          nodeSelector:
+            test_type: gpu
```

Comment:
nit: some of these have no newline at the end, while some do. Let's just be consistent across the board

Reply:
Ack